content/messaging/agent-chat-api/v3.2/rtm-reference/index.mdx

```diff
@@ -5304,6 +5304,32 @@ Informs about messages sent via the `multicast` method or by the system.
 | `content`   | Yes      |       |
 | `type`      | No       |       |
 
+### `chat_unfollowed`
+Informs that chat has been unfollowed.  Useful in multiple connections scenarios, where one connection needs to know that another one unfollowed chat.
```

Comment:
There should be an empty line after markdown header. Also, there is double space before `Useful`, it only forces a line break when the line ends with it.

Reply:
There was no extra lines after other pushes so I'm staying consistent.  I'll remove the extra space.

Reply:
It's probably my fault - I'm inconsistent when it comes to new lines after headers, so both of you might be right. Let's stick to the _no-line-after-header_ rule for the time being since that's the case for other pushes.

Reply:
@lpmichalik this is inconsistent across the docs, and both ways seem to work fine, but the missing empty line is a markdown error, just saying...